project/Dependencies.scala

```diff
@@ -17,12 +17,12 @@
 import sbt._
 
 object Version {
-  val geotools    = "25.0"
+  val geotools    = "25.4"
```

Comment:
I could not upgrade it further, see: https://github.com/locationtech/geotrellis/issues/3470